src/utils/src/time.rs

```diff
@@ -5,6 +5,8 @@ use std::fmt;
 
 /// Constant to convert seconds to nanoseconds.
 pub const NANOS_PER_SECOND: u64 = 1_000_000_000;
```

Comment:
Do we need this constant anymore?

Reply:
Yes, it is used in the `seconds_to_nanoseconds()` function.